### PR TITLE
MultiMixEnv: Adds support for make

### PR DESCRIPTION
**Pull request by Tezirg**

### CODE REVIEW

File: grid2op/MakeEnv/Make.py [hunk 1]
```diff
@@ -154,6 +159,17 @@ def _extract_ds_name(dataset_path):
     dataset_name = os.path.splitext(dataset_name)[0]
     return dataset_name
 
+def _aux_is_multimix(dataset):
+    ds_name = os.path.basename(dataset)
+    if ds_name in MULTIMIX_ENVS:
```
Comment (BDonnot) [inline grid2op/MakeEnv/Make.py]: not a fan at all of this. This re introduce a coupling between the env release and the dataset release

**Review by BDonnot — CHANGES_REQUESTED** [review]
I think there are too much data in the data/l2rpn_neurips_2020_track2 (2 months per generation is sufficient imho).

Also, it would be nice to have something different to check if an env is a multimix or not. For example, we could set up a meta file in the env folder, if this file exist it specifies the env class, if not then Environment is used.
We could also imagine a file named like `__multimix` present in folder for multimix env and check whether or not this file is present in order to do the multimix constructor.

File: grid2op/MakeEnv/Make.py (hunk 1 above)
Comment (Tezirg) [reply grid2op/MakeEnv/Make.py]: Solved by using the meta file solution.

**Comment by Tezirg** [comment]
Updated to use a metadata file instead of hardcoding multimix env names.
Dev data set `l2rpn_neurips_2020_track2` contains 2x2 chronics of 3 days for a total of 1.5Mb

File: grid2op/MakeEnv/Make.py [hunk 2]
```diff
@@ -159,9 +155,10 @@ def _extract_ds_name(dataset_path):
...
 
-def _aux_is_multimix(dataset):
-    ds_name = os.path.basename(dataset)
-    if ds_name in MULTIMIX_ENVS:
+_MULTIMIX_FILE = ".multimix"
+
+def _aux_is_multimix(dataset_path):
+    if os.path.exists(os.path.join(dataset_path, _MULTIMIX_FILE)):
         return True
     return False
```
Comment (BDonnot) [inline grid2op/MakeEnv/Make.py]: `return os.path.exists(os.path.join(dataset_path, _MULTIMIX_FILE))` would probably be less confusing, but this works too ;-)

**Review by BDonnot — APPROVED** [review]

**Pull request merged**